python/private/py_repositories.bzl

```diff
@@ -65,4 +65,10 @@ def py_repositories():
         strip_prefix = "protobuf-27.0",
         url = "https://github.com/protocolbuffers/protobuf/releases/download/v27.0/protobuf-27.0.tar.gz",
     )
+    http_archive(
+        name = "bazel_features",
+        sha256 = "c2596994cf63513bd44180411a4ac3ae95d32bf59148fcb6087a4642b3ffef11",
+        strip_prefix = "bazel_features-1.20.0",
+        url = "https://github.com/bazel-contrib/bazel_features/releases/download/v1.20.0/bazel_features-v1.20.0.tar.gz",
+    )
```

Comment:
I need to also run
```
load("@bazel_features//:deps.bzl", "bazel_features_deps")
bazel_features_deps()
```

but it's no longer possible to do an inline load call, what's the right way to accomplish this?

Reply:
#2274 also faced the same issue and we decided to backport the detection to our internal configuration repository rule. Would it be possible to do something similar in here too?

Otherwise, see what was proposed in there - adding an extra function `py_repositories_deps` that each user would have to call.

Another solution would be to only support the spaces in `bzlmod`, but that is not ideal either.

Reply:
I re-implemented this to avoid the need for bazel_features.

Given that we're stuck with workspace for another year, we probably want to introduce a `py_repositories_deps()` call for workspace, but we can punt that to a separate PR.